.github/workflows/develop.yml

```diff
@@ -37,3 +37,61 @@ jobs:
           if git add --update && git commit --no-edit --allow-empty --message "Set Version: $(cat package/version)"; then
             git push origin master
           fi
+
+  post-performance-tests:
```

Comment:
I think the goal with performance to start is to start cataloging our progress, if we fail on this in anyway, command fails, connection to bencher causes an error we don't want this to stop our release process. 

We should consider adding `continue-on-error: true`